cameo/core/manipulation.py

```diff
@@ -84,6 +84,36 @@ def decrease_flux(reaction, ref_value, value, time_machine=None):
         reaction.knock_out(time_machine=time_machine)
 
 
+def reverse_flux(reaction, ref_value, value, time_machine=None):
+    """
+    lb                           0                           ub
```

Comment:
wouldn't hurt with a couple of words like 'change the direction of the bounds on flux with respect to a reference value' or something like that. the ascii diagram isn't super intuitive...